pages/data-design/avo-tracking-plan/properties.mdx

```diff
@@ -181,17 +181,25 @@ For configuring per-event and per-source you can change the overall setting for
 Properties of type string can have rules that require the property to be a certain value or one of allowed values.
 When these rules are applied on the property, the generated avo code will make sure that these values are only being sent.
 
-![Image shows how allowed property values enumeration](/images/property_enum.png)
+![Image shows how allowed values on a property](/images/property_enum.png)
 
-#### Source specific allowed values
+#### Event and source specific allowed values
 
 <Callout type="info">
-  Source specific property rules are available on the Team plan.
+  Event and source specific property rules are available on the Team plan.
 </Callout>
 
-When a property is sent from multiple sources, you can configure from which sources each value is allowed to be sent.
+When a property is sent from multiple events and sources, you can configure the allowed values granularly for each event and each source.
 
-![Image shows how allowed property values can be enabled or disabled on each source](/images/property_enum_source_specific.png)
+It can be configured both from the property details
+
+![Image shows how allowed property values can be enabled or disabled on each event and source from property details](/images/property_enum_edit_in_property.png)
```

Comment:
We should update the screenshots with better demo content, something people can more easily relate to. I've migrated the Demo Tracking Plan, perhaps we can set up a case there to grab some screenshots?